From f624ba457de24def838ecf4e5fe53a0ad9f35d57 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Wed, 29 Jul 2020 11:44:52 +0800
Subject: [PATCH] Test case5 for Add module in component&library class

Description: Add the library module (MdeModulePkg\Library\BaseSortLib\BaseSortLib.inf) in component (Sec) section and library class.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/LibraryClass.dsc | 1 +
 TestPkg/Sec/SecMain.inf  | 1 +
 TestPkg/TestPkg32X64.dsc | 1 +
 3 files changed, 3 insertions(+)

diff --git a/TestPkg/LibraryClass.dsc b/TestPkg/LibraryClass.dsc
index 6530bafbe7..c3b63cd685 100644
--- a/TestPkg/LibraryClass.dsc
+++ b/TestPkg/LibraryClass.dsc
@@ -44,6 +44,7 @@ NvVarsFileLib|TestPkg/Library/NvVarsFileLib/NvVarsFileLib.inf
   MtrrLib|UefiCpuPkg/Library/MtrrLib/MtrrLib.inf
   FakeLibraryLib|TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf
   LockBoxLib|MdeModulePkg\Library\LockBoxNullLib\LockBoxNullLib.inf
+  SortLib|MdeModulePkg\Library\BaseSortLib\BaseSortLib.inf
 
 [LibraryClasses.common.SEC]
   TimerLib|TestPkg/Library/AcpiTimerLib/BaseRomAcpiTimerLib.inf
diff --git a/TestPkg/Sec/SecMain.inf b/TestPkg/Sec/SecMain.inf
index 3066e0d102..a8f073f581 100644
--- a/TestPkg/Sec/SecMain.inf
+++ b/TestPkg/Sec/SecMain.inf
@@ -52,6 +52,7 @@
   LocalApicLib
   FakeLibraryLib
   LockBoxLib
+  SortLib
 
 [Ppis]
   gEfiTemporaryRamSupportPpiGuid                # PPI ALWAYS_PRODUCED
diff --git a/TestPkg/TestPkg32X64.dsc b/TestPkg/TestPkg32X64.dsc
index b24f19fc5f..4e2d1693eb 100644
--- a/TestPkg/TestPkg32X64.dsc
+++ b/TestPkg/TestPkg32X64.dsc
@@ -118,6 +118,7 @@
     <LibraryClasses>
     NULL|MdeModulePkg/Library/LzmaCustomDecompressLib/LzmaCustomDecompressLib.inf
 	LockBoxLib|MdeModulePkg\Library\LockBoxNullLib\LockBoxNullLib.inf
+	SortLib|MdeModulePkg\Library\BaseSortLib\BaseSortLib.inf
   }
 
   #
-- 
2.27.0.windows.1

